test/onnx/exporter/test_compat.py

```diff
@@ -6,6 +6,27 @@
 import torch
 from torch.onnx._internal.exporter import _compat
 from torch.testing._internal import common_utils
+from torch.utils import _pytree
+
+
+class LlamaModelTest(torch.nn.Module):
```

Comment:
```suggestion
class LlamaModel(torch.nn.Module):
```

otherwise it gets picked up as a test class